python/sqlalchemy/src/rest-service.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import logging
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import json
+from io import BytesIO
+import jsonpickle
+from yugabyte_alchemy_model import User, Order, OrderLine, Product
+
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s:%(levelname)s:%(message)s"
+    )
+
+logging.getLogger('sqlalchemy.engine.base.Engine').setLevel(logging.WARNING)
+
+listen_port = 8000
+db_user = 'postgres'
+db_password = 'pwd'
+database = 'yugabyte_alchemy'
+db_host = 'localhost'
+db_port = 5433
+
+from sqlalchemy import create_engine
+engine = create_engine('postgresql://%s:%s@%s:%s/%s' % 
+                       (db_user, db_password, db_host, db_port, database), echo=True)
+Session = sessionmaker(bind=engine)
+
+
+def add_object(session, to_add):
+    session.add(to_add)
+    logging.debug('Added: %s' % str(to_add) )
+
+
+def add_objects(session, object_list):
+    session.add_all(object_list)
+    logging.debug('Added (list): ')
+    [logging.debug(str(obj)) for obj in object_list]
+
+class DataAccessUtil:
```

Comment:
Oh sorry got lost in git reviews. This name is good! Ignore my other comment.

Can we move this class into a separate file with a similar name?

Reply:
Moved.  Done.